src/ui/server/controllers/insurance/policy/save-data/nominated-loss-payee/index.ts

```diff
@@ -1,8 +1,19 @@
 import api from '../../../../../api';
 import getDataToSave from '../../../../../helpers/get-data-to-save';
 import { sanitiseData } from '../../../../../helpers/sanitise-data';
+import stripEmptyFormFields from '../../../../../helpers/strip-empty-form-fields';
 import { Application, RequestBody } from '../../../../../../types';
 
+import POLICY_FIELD_IDS from '../../../../../constants/field-ids/insurance/policy';
+
+const { NAME } = POLICY_FIELD_IDS.LOSS_PAYEE_DETAILS;
+
+/**
+ * string fields which are exempt from being stripped by stripEmptyFormFields
+ * for example when a string field needs to be set to an empty string or null
+ */
+export const NULL_OR_EMPTY_STRING_FIELDS = [NAME];
```

Comment:
Why is `NAME` exepmt from being set to empty/null - what happens if a user removes all text from the input and submits?

Reply:
if you remove text from NAME and press save and back, it won't save your changes (remove the value for NAME) without this exemption